internal/config/generate/Config.toml

```diff
@@ -0,0 +1,215 @@
+# (c) Cartesi and individual authors (see AUTHORS)
+# SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+# ----------------------------------------
+# Auth -----------------------------------
+# ----------------------------------------
```

Comment:
```suggestion
#
# Auth
#
```

Even though it looks nice, I'd rather avoid maintaining this ASCII art.

Reply:
Not a patron of the arts...

 Let's discuss dividers today then. I think it is crucial to be able to divide code visually.